src/signframework/sign_cert/tests/issuer_final_sample.cfg

```diff
@@ -0,0 +1,40 @@
+# final CA project configuration file
+
+# NOTE: The parameters must be in this order!
+#
+#	program executable filename
+
+program=sign_cert/sign_cert
+
+# File name for the project audit log
+
+log=sign_cert/tests/issuer_final.log
+
+#	project signing key file name
+
+needkey=true
+key=sign_cert/tests/issuer_final.tok
+
+#	auxiliary project configuration file
+
+needauxcfg=true
+auxcfg=sign_cert/tests/issuer_final_sample_aux.cfg
+
+neediatt=false
+needoatt=true
+needpwd=true
+
+#	Project administrator email address
+
+email=nick.child@ibm.com
+
+#	Notification receivers
+
+notifs=0
+
+#       Authorized senders (signers)
+
+needsenders=true
+senders=1
+nnac=nick.child@ibm.com
```

Comment:
You may want to remove your email/id

The other sample configs use fred=fred@example.com